notebooks/02-datamodules.ipynb

```diff
@@ -403,10 +402,9 @@
         "        loss = F.nll_loss(logits, y)\n",
         "        preds = torch.argmax(logits, dim=1)\n",
         "        acc = accuracy(preds, y)\n",
-        "        result = pl.EvalResult(checkpoint_on=loss)\n",
-        "        result.log('val_loss', loss, prog_bar=True)\n",
```

Comment:
note that `self.log` won't work until theres a stable release of lightning w/ that functionality. At top of these notebooks we are doing simple `pip install pytorch-lightning`, so its not picking up the prerelease. I think that's fine if we intend to release 0.10.0 today, but these notebooks will all be broken until then otherwise.

Reply:
yes, I was installing it with `pip install pytorch-lightning --pre` but we do not want to have it in notebooks, right?